easybuild/easyconfigs/i/impi/impi-2019.7.217-iccifort-2020.1.217.eb

```diff
@@ -12,11 +12,24 @@ source_urls = ['https://registrationcenter-download.intel.com/akdlm/irc_nas/tec/
 sources = ['l_mpi_%(version)s.tgz']
 checksums = ['90383b0023f84ac003a55d8bb29dbcf0c639f43a25a2d8d8698a16e770ac9c07']
 
+dependencies = [
+  # Uncomment this if your system UCX is too old.
+  #    ('UCX', '1.8.0'),  # needed by libfabric provider MLX introduced in Intel MPI v2019.6
```

Comment:
Maybe we should just include that by default?

Is there a downside to providing UCX through EasyBuild?

Reply:
Maybe some sites prefer to use UCX from HPCx, but I don't see many downsides otherwise.

In any case that's waiting for #10329

Reply:
I prefer to have UCX added by default, even if UCX is available in the host system a too lower version will cause issues.